boards/nordic/nrf54h20dk/board.cmake

```diff
@@ -3,7 +3,8 @@
 include(${ZEPHYR_BASE}/boards/common/nrfutil.board.cmake)
 
 if(CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUAPP OR CONFIG_BOARD_NRF54H20DK_NRF54H20_CPURAD
-    OR CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON)
+    OR CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON
+    OR CONFIG_BOARD_NRF54H20DK_NRF54H20_CPURAD_IRON)
```

Comment:
This formatting makes my eyes bleed

Reply:
You want one option on each line right?

Reply:
`OR` on end-of-line, perhaps? CONFIG_ on same vertical?

Reply:
Yeah one per line would be nice instead of two on one then one on each of the next.

Sorry for the super-unclear comment :D (not sarcastic)

Reply:
fixed